marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mapping/default/lib.sjs

```diff
@@ -15,13 +15,13 @@ function getModel(targetEntity, version = '0.0.1') {
 }
 
 function getMapping(mappingName) {
-  return fn.head(cts.search(cts.andQuery([cts.collectionQuery('http://marklogic.com/data-hub/mappings'), cts.jsonPropertyValueQuery('name', mappingName, ['case-insensitive'])]), ["unfiltered", cts.indexOrder(cts.uriReference(), "descending")]));
+  return fn.head(cts.search(cts.andQuery([cts.collectionQuery('http://marklogic.com/data-hub/mappings'), cts.jsonPropertyValueQuery('name', mappingName, ['unstemmed', 'case-insensitive'])]), ["unfiltered", cts.indexOrder(cts.uriReference(), "descending")]));
```

Comment:
Related to this is that we have a lot of cts.search functions that perform filtered searches when they don't need to. I am thinking a convenience function would be handy here that defaults to unfiltered searches. And to handle the common use case of "Find me an artifact with this name in this collection", it would have a signature of:

    findArtifact(artifactName, artifactCollection)

That would likely be in artifacts/core.sjs since it's specific to artifacts.

hub-utils could have a more general search function - e.g. 

    hubUtils.findDocuments(myQuery, optionalIndexOrder)

That function would default to an unfiltered search.